app/models/tasks/decision_review_task.rb

```diff
@@ -60,4 +62,11 @@ def validate_decision_issue_per_request_issue(decision_issue_params)
       decision_issue_param[:request_issue_id].to_i
     end.sort
   end
+
+  def validate_decision_date(decision_date)
+    return true unless assigned_to == VhaBusinessLine.singleton
```

Comment:
I think using is_a? is a bit more idiomatically correct for Ruby here. 
```suggestion
    return true unless assigned_to.is_a?(VhaBusinessLine)
```